src/Adliance.AspNetCore.Buddy.OpenTelemetry/Extensions/BuddyApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.FileProviders;
+
+namespace Adliance.AspNetCore.Buddy.OpenTelemetry.Extensions;
+
+public static class BuddyApplicationBuilderExtensions
+{
+    /// <summary>
+    /// Adds a static file server for /otel-js/telemetry.js.
+    /// </summary>
+    /// <param name="applicationBuilder">The application builder.</param>
+    /// <returns>The application builder to allow further calls.</returns>
+    public static IApplicationBuilder AddBuddyOpenTelemetryBrowserAssets(
+        this IApplicationBuilder applicationBuilder)
+    {
+        var assembly = typeof(BuddyApplicationBuilderExtensions).GetTypeInfo().Assembly;
+        var embeddedFileProvider = new EmbeddedFileProvider(
+            assembly,
+            "Adliance.AspNetCore.Buddy.OpenTelemetry.wwwroot.lib.dist"
+        );
+        
+        applicationBuilder.UseStaticFiles(new StaticFileOptions
```

Comment:
Hmm, I'm not incredibely happy with it, because this way the user of the library cannot choose NOT to use static files. Can we maybe use a custom middleware for that instead of using the static files middleware? Or maybe you have a better idea? Also, please set a good cache header for the JS file.